### ISSUE TITLE
Talkback doesn't read new status for DialView on click

### ISSUE THREAD

**Issue by Tobino** [googlesamples/android-BasicAccessibility#2]
When a user click on DialView with TalkBack, TalkBack should read the new status "Mode selected: X."

It seem the event is well populate, but TalkBack ignore change.
To get the new status, you have to leave and re-enter the customView to get the new status.

**Comment by codingjeremy** [comment]
This sample has been deprecated/archived (check README for more information on newer samples related to this technology). 

[As recommended by GitHub][1], we are closing all issues and pull requests.

[1]: https://help.github.com/en/articles/archiving-repositories
